packages/frontend-shared/src/gql-components/HeaderBar.vue

```diff
@@ -2,8 +2,9 @@
   <HeaderBarContent
     v-if="query.data.value"
     :gql="query.data.value"
-    :show-browsers="props.showBrowsers"
+    :show-browsers="showBrowsers"
```

Comment:
Not really related to this PR but is there a specific reason we don't like to refer to props as `props.`?

I prefer to use `props.` but either way we should lint for one of the other.

Let's add this to our list of "to lint", including

- enforce using `props.` (or not using `props.`)
- kebab vs camel case

Reply:
Agreed, no strong preference here, would rather lint fix and forget about :)